src/main/java/com/microfocus/octane/plugins/configuration/ConfigurationManager.java

```diff
@@ -182,10 +182,12 @@ public synchronized void clearConfiguration() {
 
     private ConfigurationCollection loadConfiguration() {
         PluginSettings settings = pluginSettingsFactory.createGlobalSettings();
-        String confStr = ((String) settings.get(CONFIGURATION_KEY_V2));
+        String confStr = readConfigurationForDataKey(settings, CONFIGURATION_KEY_V3);
         log.info("Configuration is loading...");
-        if (confStr == null) {//create initial configuration
-            configuration = tryConvertFromPreviousVersion(settings);
+
+        if (confStr == null) {
```

Comment:
C: use isBlank from String utils. its safer cause it also checks if string is empty

Reply:
in this case, I don't really care about empty string, I want to check if previous configuration exists at all in the jira server
if it's empty string it's a different case - it existed, but it was empty